forge-config-scripts/scripts/ReceiveConfig.s.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
```

Comment:
any reason you used this version?  Most OFT code uses 0.8.22

Reply:
Nice catch -- just a copy paste error!